packages/web-component/src/lib/descope-wc/BaseDescopeWc.ts

```diff
@@ -55,6 +55,22 @@ class BaseDescopeWc extends HTMLElement {
 
   #init = false;
 
+  #log = {
```

Comment:
I didn't understand the inheritance pattern where log is implemented both in `BaseDescopeWc` and in `DescopeWc`

why do we need it? 

I don't think that logger in the base should trigger `next` callback (aka dispatchNext)

might be simpler to make it as before

Reply:
since its an object i moved kept it a property and the logger became a function so we can override it in the DescopeWc